src/Components/Onboarding.js

```diff
@@ -38,8 +38,10 @@ export default function Onboarding() {
   const navigate = useNavigate();
 
   useEffect(() => {
-    //Check for existing user auth
-    if (user) return navigate("/home");
+    if (!user) return;
+    // Let anonymous users register accounts and
+    // redirect existing authorized users
+    if (!user?.isAnonymous) return navigate("/home");
```

Comment:
`useEffect` functions return value should be a function that will be invoked as a clean-up the next time the `useEffect` function gets run.  So, here, instead of `return navigate("/home");`, you can probably just do `navigate("/home")` and let the function return undefined implicitly.

Also, nit: if we know user has value here (since there was an early return above if it didn't), then the `?` is not needed.

Reply:
Great comments!!